server/clip_server/model/pretrained_models.py

```diff
@@ -143,12 +145,30 @@ def get_model_url_md5(name: str):
     if len(model_pretrained) == 0:  # not on s3
         return None, None
     else:
+        hg_download_url = (
+            _OPENCLIP_HUGGINGFACE_BUCKET
+            + 'resolve/main/'
+            + model_pretrained[0]
+            + '?download=true'
+        )
+        try:
+            print(f'Test Hg url: {hg_download_url}')
```

Comment:
Remove the test print. Raise whenever necessary

Reply:
Okay, the test print has been removed and the onnx model downloader address handled by now.